lighthouse-cli/test/fixtures/form.html

```diff
@@ -0,0 +1,122 @@
+<html><head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Autofill Smoke Test</title>
+</head>
+<body>
+  <form id="checkout" name="checkout" action="../done.html" method="post">
+      <label id="cardName" for="name_cc">Name on card:</label> <textarea type="text" name="name_cc" autocomplete="cc-name"></textarea><br>
```

Comment:
Are these labels intentionally not associated with their inputs? If so, it probably deserves a comment to assure the reader it's not a typo.

(or typo fix them)
```suggestion
      <label id="cardName" for="name_cc">Name on card:</label> <textarea type="text" id="name_cc" autocomplete="cc-name"></textarea><br>
```